src/classes/RD2_OpportunityEvaluationService.cls

```diff
@@ -213,6 +213,36 @@ public inherited sharing class RD2_OpportunityEvaluationService {
         } set;
     }
 
+    /**
+    * @description Contains all necessary fields for Recurring Donations Evaluation Services 
+    */
+    private Set<String> rdFields = new Set<String> {
+        'npe03__Organization__c',
```

Comment:
- `@description Contains all mandatory Recurring Donation fields required for the Opportunity Evaluation Service`
- Let's add Name field to the field list as well.